devops/scripts/deploy_shared_service.sh

```diff
@@ -59,22 +60,23 @@ if [[ -n "${deployed_shared_service}" ]]; then
     echo "Shared service ${template_name} of version ${template_version} has already been deployed"
     exit 0
   else
-    echo "Resource upgrade isn't currently implemented. See https://github.com/microsoft/AzureTRE/issues/141"
-    exit 0
+    is_update=1
   fi
 fi
 
-# Add additional properties to the payload JSON string
-additional_props=""
-for index in "${!property_names[@]}"; do
-  name=${property_names[$index]}
-  value=${property_values[$index]}
-  additional_props="$additional_props, \"$name\": \"$value\""
-done
+if [[ "${is_update}" -eq 0 ]]; then
+
+  # Add additional properties to the payload JSON string
+  additional_props=""
+  for index in "${!property_names[@]}"; do
+    name=${property_names[$index]}
+    value=${property_values[$index]}
+    additional_props="$additional_props, \"$name\": \"$value\""
+  done
 
-echo "Not currently deployed - deploying..."
-display_name="${template_name#tre-shared-service-}"
-if ! deploy_result=$(cat << EOF | tre shared-services new --definition-file -
+  echo "Not currently deployed - deploying..."
+  display_name="${template_name#tre-shared-service-}"
+  if ! deploy_result=$(cat << EOF | tre shared-services new --definition-file -
```

Comment:
I think the indent here is incorrect

Reply:
I understand why you think that but due to the EOF is has to be like that.